lib/matplotlib/tests/test_afm.py

```diff
@@ -0,0 +1,13 @@
+import matplotlib.afm as afm
```

Comment:
I've never seen a static checker complain about `__future__` imports; how would they even know if you used, e.g., `division` with types that were integers?

This is about consistent Python 2/3 behaviour, not some random import.